### PR TITLE
update codeowners

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
changed everything to use teams (except sprites theres no sprite team :trollface:)
- sprites reviewed by velcroboy
- map files and the map test get reviewed by maptainers team
- yml files get reviewed by yml maintainers team
- code in any assembly gets reviewed by maintainers team
- lobby art and music get reviewed by direction team

notably this makes the colin review on Resources/*.yml actually work as intended since thats not recursive, it wouldve only triggered for keybinds prs :trollface:

### CODE REVIEW

**Review by Lyndomen — APPROVED** [review]
velcro spriteguy real

**Comment by IamVelcroboy** [comment]
> velcro spriteguy real

Woah

**Comment by Samsterious** [comment]
Art Direction would presumably go under Game Direction in the same way that Mapping does, but *shrug, close enough. 👍

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
-/Resources/Prototypes/Maps/ @IamVelcroboy
+# YML files
+/Resources/**.yml @DeltaV-Station/yaml-maintainers
+
+# Sprites
+/Resources/Textures/ @IamVelcroboy
+
+# Lobby art and music - automatically direction issues since its immediately visible to players
+/Resources/Audio/Lobby/ @DeltaV-Station/game-directors
+/Resources/Textures/LobbyScreens/ @DeltaV-Station/game-directors
```
Comment (Samsterious) [inline .github/CODEOWNERS:14]: According to Github, `@DeltaV-Station/game-directors` does not exist.

**Comment by deltanedas** [comment]
@DeltaV-Station/game-directors real or fake

File: .github/CODEOWNERS (hunk 1 above)
Comment (deltanedas) [reply .github/CODEOWNERS:14]: ok it was just not very specific, you need write access not triage or it wont let you be a codeowner :trollface:

Comment (Samsterious) [reply .github/CODEOWNERS:14]: Not sure why the code is flagged as erroneous during review, then. Ah well.

**Review by Samsterious — APPROVED** [review]

**Pull request merged**